lesson12/script.js

```diff
@@ -9,21 +9,23 @@ function removeLoadingImg(){
     document.getElementById('js-loading').remove();
 }
 
-
+async function fetchdata(){
+    const endpoint = "https://api.json-generator.com/templates/szdgGQcOLXuk/data?access_token=hu4bc7qh9znx2m8f53mn4mz2hryvdntkavwbw8j0";
+    const response = await fetch(endpoint);
+    //responseがokではないとき
+    if(!response.ok){
+    throw new Error(`${response.status}:${response.text}`);
+    }
+    //responseがokのとき
+    const json = await response.json();
+    const data = await json.data;
+    return data;
+}
 
 async function getData(){
     showLoadingImg();
     try{
-        const endpoint = "https://api.json-generator.com/templates/szdgGQcOLXuk/data?access_token=hu4bc7qh9znx2m8f53mn4mz2hryvdntkavwbw8j0";
-        const response = await fetch(endpoint);
-        //responseがokではないとき
-        if(!response.ok){
-        throw new Error(`${response.status}:${response.text}`);
-        }
-        //responseがokのとき
-        const json = await response.json();
-        const data = await json.data;
-        return data;
+        return await fetchdata();
```

Comment:
The process of fetching the data was separated into functions, 
so I added fetchdata ().